### PR TITLE
Update construction_time.cpp

### PR DESCRIPTION
**Pull request by ajwahab**
Corrected several type typos and adjusted indentation.

### CODE REVIEW

File: .theia/launch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (crvs) [inline .theia/launch.json]: this file seems to be from your local build system, can you remove or from the commit?

File: .theia/settings.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (crvs) [inline .theia/settings.json]: same

File: .theia/tasks.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=733558
```
Comment (crvs) [inline .theia/tasks.json]: same

File: CMakeLists.txt [hunk 4]
```diff
@@ -1,26 +1,26 @@
 cmake_minimum_required(VERSION 2.8)
 project(KDTree CXX)
 
-set(CMAKE_CXX_COMPILER             "/usr/bin/g++")
```
Comment (crvs) [inline CMakeLists.txt:4]: maybe this shouldn't be hard coded then...

File: tests/construction_time.cpp [hunk 5]
```diff
@@ -20,32 +20,32 @@ std::vector<double> generateVector()
...
+    for (size_t idx = 0; idx < length; idx++)
     {
-		temp[idx] = generateVector();
+        temp[idx] = generateVector();
     }
-	return temp;
+    return temp;
 }
 
-snt main()
```
Comment (crvs) [inline tests/construction_time.cpp:38]: how did that compile?!

**Review by crvs — CHANGES_REQUESTED** [review]

**Comment by ajwahab** [comment]
Oh man, sorry for all the mess - I only intended to merge changes to construction_time.cpp.

File: CMakeLists.txt [hunk 6]
```diff
@@ -1,26 +0,0 @@
-cmake_minimum_required(VERSION 2.8)
```
Comment (crvs) [inline CMakeLists.txt:1]: I think this wasn't supposed to be deleted

File: tests/CMakeLists.txt [hunk 7]
```diff
@@ -1,9 +0,0 @@
-
```
Comment (crvs) [inline tests/CMakeLists.txt:1]: ...and this

File: tests/toy_test.cpp [hunk 8]
```diff
@@ -1,78 +0,0 @@
-#include <iostream>
```
Comment (crvs) [inline tests/toy_test.cpp:1]: ... and this

**Comment by crvs** [comment]
No problem, I think you just accidentally deleted a few files too many

**Pull request closed without merging**